Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -75,6 +78,58 @@ async function main(): Promise<void> {
         var artifacts = [];
         var itemPattern: string = '**';
 
+        var releaseAlias: string = tl.getVariable("release.triggeringartifact.alias");
+
+        if (isCurrentBuild) {
+            projectId = tl.getVariable("System.TeamProjectId");
+            definitionId = '';
+            buildId = parseInt(tl.getVariable("Build.BuildId"));
+        }
+        else{
+            let releaseUri = tl.getVariable("Release.ReleaseUri");
```

Comment:
use "system.hosttype" for build it's value will be "build", for release it can be "release" or "deployment"